modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/SystemObjectRelatedObjectsTest.java

```diff
@@ -194,6 +209,177 @@ public void testGetOneToManySystemObjectRelatedObjects() throws Exception {
 			_objectEntry);
 	}
 
+	@Test
+	public void testPostSystemObjectEntryWithInvalidNestedCustomObjectEntries()
+		throws Exception {
+
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				"com.liferay.portal.vulcan.internal.jaxrs.exception.mapper." +
+					"WebApplicationExceptionMapper",
+				LoggerTestUtil.WARN)) {
+
+			ObjectRelationship objectRelationship =
+				ObjectRelationshipTestUtil.addObjectRelationship(
+					_userSystemObjectDefinition, _objectDefinition,
+					_user.getUserId(),
+					ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+			_objectRelationships.add(objectRelationship);
+
+			_testPostSystemObjectEntryWithInvalidNestedCustomObjectEntriesInManyToManyRelationship(
+				objectRelationship);
+
+			_testPostSystemObjectEntryWithInvalidNestedCustomObjectEntriesInOneToManyRelationship(
+				objectRelationship);
```

Comment:
This method says that is for OneToMany relationships but the relationship that you are providing is a many-to-many relationship, isn't it? Is that correct? Am I misunderstanding something? 😅

Reply:
You are right @LuismiBarcos!! My fault :sweat_smile: